python/src/spark_rapids_ml/core.py

```diff
@@ -1306,6 +1304,12 @@ def _pre_process_data(
                     raise ValueError(
                         "All columns must be integral types or float/double types."
                     )
+
+            taglen = len("_c3BhcmstcmFwaWRzLW1sCg==")
```

Comment:
Better to define the tag in a new variable so all references to it are consistent.